src/GlobalStateProvider.tsx

```diff
@@ -3,13 +3,13 @@ import { createContext } from 'preact';
 import { useCallback, useContext, useEffect, useMemo, useRef, useState } from 'preact/compat';
 import { useLocation } from 'react-router-dom';
 import { config } from './config';
+import { chainIds } from './config/walletConnect';
```

Comment:
Do we need this import?

Reply:
I see this is indeed used in [line 97](https://github.com/pendulum-chain/portal/blob/ad6735de6f05dcecb2cb5c3d9c730775d8d6b232/src/GlobalStateProvider.tsx#L97) in the same file, I think we can resolve this conv and merge?